functions/auth.go

```diff
@@ -14,19 +13,15 @@ import (
 )
 
 // Authenticate authenticates with netmaker api to permit subsequent interactions with the api
-func Authenticate(node *config.Node, host *config.Config) (string, error) {
+func Authenticate(url, network string, host *config.Config) (string, error) {
 	data := models.AuthParams{
 		MacAddress: host.MacAddress.String(),
-		ID:         node.ID.String(),
+		ID:         host.ID.String(),
 		Password:   host.HostPass,
 	}
-	server := config.GetServer(node.Server)
-	if server == nil {
-		return "", errors.New("nil server")
-	}
 	endpoint := httpclient.Endpoint{
-		URL:    "https://" + server.API,
-		Route:  "/api/nodes/adm/" + node.Network + "/authenticate",
+		URL:    "https://" + url,
+		Route:  "/api/hosts/adm/" + network + "/authenticate",
```

Comment:
host auth shouldn't be associated with network

Reply:
but network is required for creating JWT
```
tokenString, err := logic.CreateJWT(authRequest.ID, authRequest.MacAddress, mux.Vars(request)["network"])
```

Reply:
we should remove need for network specification for host auth

Reply:
done